.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+disabled_rules=import-ordering
```

Comment:
What tool does this affect, ktlint? Why do we _not_ want our imports sorted?

Reply:
Yes, Ktlint. Removed.